src/core/server/logging/appenders/rolling_file/strategies/index.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { RollingStrategy } from './strategy';
+import {
+  NumericRollingStrategy,
+  NumericRollingStrategyConfig,
+  numericRollingStrategyConfigSchema,
+} from './numeric';
+import { RollingFileContext } from '../rolling_file_context';
+
+export { RollingStrategy } from './strategy';
+export type RollingStrategyConfig = NumericRollingStrategyConfig;
+
+export const rollingStrategyConfigSchema = schema.oneOf([numericRollingStrategyConfigSchema]);
```

Comment:
Since we only have a single strategy available, what do you think about providing a default value, so that consumers aren't forced to specify it? I mistakenly omitted this from my yml, and the `@kbn/config-schema` error message was hard to decipher at best.

Reply:
Yea, I even added default values for both the policy and the strategy at some point, so that

```yaml
rolling-file:
    kind: rolling-file
    path: /Users/pierregayvallet/Documents/test-log/kibana.log
    layout:
        kind: pattern
```

would be enough, then I reverted it, thinking that we didn't do that for other appenders (like you must always specify `layout`, there isn't a default value for it).

@elastic/kibana-core do you have any opinion on this?